widget/embedded/src/components/Layout/Layout.styles.ts

```diff
@@ -1,52 +1,44 @@
 import { darkTheme, styled } from '@rango-dev/ui';
 
+const WIDGET_HEIGHT = '700px';
 export const Container = styled('div', {
   position: 'relative',
   display: 'flex',
   flexDirection: 'column',
   borderRadius: '$primary',
-  overflow: 'hidden !important',
-  boxShadow: '15px 15px 15px 0px rgba(0, 0, 0, 0.05)',
```

Comment:
the boxShadow has been removed. Don't we need it anymore?

Reply:
I wanted to move it somewhere else but seems removed it completely by mistake.

Reply:
fixed.